src/components/BlogPost.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import weekendfly from '../images/weekendfly.jpg';
+
```

Comment:
Great job getting varied content into the blogspots. You'll want to start styling it in this file. I recommend using Grid to style these blogpost. Try using a grid container with two grid items inside. one item for date & genre (you can leave out "leave a comment") and another item with your content. Reference Page.js on how it should look like.